Tournament_Calculator.py

```diff
@@ -0,0 +1,323 @@
+#import modules
+import openpyxl
+import random
+import re
+
+#Player Name/Hero dictionary and set variable for below function
+y=0
+Player_List = {}
+
+
+#Function to populate dictionary with player name/Hero
+def Player_Creation(Hero):
+    global y
+    for row in sheet_obj.iter_rows(1):
+        for cell in row:
+            if cell.value == Hero:
+                z = (sheet_obj.cell(row=(cell.row), column=(2)).value)
+                x = 0
+                if z == 0:
+                    pass
+                else:
+                    while x < int(z):
+                        New_Player = "Player" + str(y)
+                        Player_List[New_Player] = Hero
+                        y +=1
+                        x +=1
+                return y
+
+ 
+
+#Set Excel defaults for player input
+path = "Input Data.xlsx"
+wb_obj = openpyxl.load_workbook(path, data_only=True)
+wb_obj.active = wb_obj['Data Input']
+sheet_obj = wb_obj.active
+
+#Populate Dictionary with Player names/Heroes
+Player_Creation("Arakni")
+Player_Creation("Azalea")
+Player_Creation("Betsy")
+Player_Creation("Bravo")
+Player_Creation("DIO")
+Player_Creation("Dash") 
+Player_Creation("Dorinthea")
+Player_Creation("Enigma")
+Player_Creation("Fai")
+Player_Creation("Kano") 
+Player_Creation("Kassai")
+Player_Creation("Katsu")
+Player_Creation("Kayo")
+Player_Creation("Levia")
+Player_Creation("Maxx")
+Player_Creation("Nuu")
+Player_Creation("Olympia")
+Player_Creation("Prism")
+Player_Creation("Rhinar") 
+Player_Creation("Riptide")
+Player_Creation("Boltyn")
+Player_Creation("Teklovossen")
+Player_Creation("Uzuri")
+Player_Creation("Victor") 
+Player_Creation("Viserai")
+Player_Creation("Vynnset")
+Player_Creation("Zen")
+
+#Set player total and rounds
+Month = ((sheet_obj.cell(row=(30),column=(2))).value)
+Players = ((sheet_obj.cell(row=(31),column=(2))).value)

```

Comment:
Another thing that would help with overall readability would be to improve the reader's understanding of what the code is supposed to produce. Two ways to do that:

1. Pad out the readme with some basic examples of a small input and what the corresponding output from the script would be
2. The code is not quite ready for this (it will be closer after this round of comments), but add some basic unit tests (github actions again) that provide some test data to the script and check that the result is correct. This has the dual benefit of automatically checking your code's correctness and also providing a self-documenting example of what the code is supposed to do. Unit tests are also one of those things that you need to know how to do well to function professionally. Once you have the script set up to take arguments and / or have your code executing from inside a function it is easier to write tests.